base/bin/build-init

```diff
@@ -7,8 +7,16 @@ set -e # Abort if anything fails
 
 # Cleanup
 echo "Setting up remote build environment..."
-ssh docker-host "(cd ${REMOTE_BUILD_DIR} 2>/dev/null && fin rm -f 2>/dev/null) || true"
-ssh docker-host "sudo rm -rf ${REMOTE_BUILD_DIR} 2>/dev/null; mkdir -p ${REMOTE_BUILD_DIR}"
+if [[ "${REMOTE_BUILD_DIR_CLEANUP}" == 1 ]] || [[ "${REMOTE_CODEBASE_METHOD}" == "rsync" ]]; then
+	echo "Re-initializing remote environment from scratch..."
+	[[ "${REMOTE_BUILD_DIR_CLEANUP}" == 0 ]] && [[ "${REMOTE_CODEBASE_METHOD}" == "rsync" ]] && echo "rsync is not supported for preserved builds!"
```

Comment:
Use either `"${REMOTE_BUILD_DIR_CLEANUP}" == "0"` (comparing strings) or `${REMOTE_BUILD_DIR_CLEANUP} == 0` (comparing numbers).

Reply:
Going with the strings method.